flink-dist/src/main/resources/flink-conf.yaml

```diff
@@ -107,7 +107,18 @@ parallelism.default: 1
 #==============================================================================
 
 # The backend that will be used to store operator state checkpoints if
-# checkpointing is enabled.
+# checkpointing is enabled. Checkpointing is enabled when execution.checkpointing.interval > 0.
+#
+# Execution checkpointing related parameters.
+#
+# execution.checkpointing.interval: -1
```

Comment:
I think setting `execution.checkpointing.interval` as `-1` is illegal as minimal checkpoint time is 10ms. Maybe we could set it as `180 s` with some comments.

Reply:
In CheckpointConfig, checkpointInterval default value is -1.  -1 means checkpoint is disabled. And I found that default value is 500ms in function enableCheckpointing().  I am confused  which value is more reasonable.

Reply:
You could refer to [CheckpointConfig#setCheckpointInterval](https://github.com/apache/flink/blob/60c7d9e77a6e9d82e0feb33f0d8bc263dddf2fd9/flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/CheckpointConfig.java#L219-L227) for more details.

Reply:
From my point of view, user might uncomment this line for out-of-box usage. However, the value of `-1` is illegal and what I thought is like
~~~ bash
# The interval in which checkpoints are periodically scheduled.
#
# execution.checkpointing.interval: 3 min
~~~

Reply:
500ms or 180s？In order to be consistent with the code, is it more appropriate to use 500ms？

Reply:
Actually, the default value of `500ms` you said really confused me, this option is a user-control value and have no default value. Where did you find the `500ms` configuration?

Reply:
You can see it here [StreamExecutionEnvironment#enableCheckpointing](https://github.com/apache/flink/blob/60c7d9e77a6e9d82e0feb33f0d8bc263dddf2fd9/flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java/#L518)

Reply:
This method is already deprecated, and `500ms` is really too quick for production users.

Reply:
Yes, I also think it's too quick. I will change it to 3min.